cmd/server/main.go

```diff
@@ -48,6 +49,11 @@ func main() {
 	)
 	flag.Parse()
 
+	if len(*keys) != len(*addrs) {
```

Comment:
now key is a required argument where before it was not, this is technically a breaking change. what happened before if key was left empty, would key-server just idle?

Reply:
it should still work since we set a default value of [""]. len(keys) == 1